### ISSUE TITLE
Error after upgrade to Statamic 5.x

### ISSUE THREAD

**Issue by plompd** [eminos/statamic-iconify#2]
With the latest version of Iconify and updated to the latest version of Statamic I get an error using the addon:

```
 [internal function]: Statamic\\Tags\\Tags->__call('index', Array)
```

**Comment by eminos** [comment]
I can't really reproduce this. Could you send more info or the full stack trace? What are you doing exactly to trigger this error?

**Comment by plompd** [comment]
Here is the trace: https://flareapp.io/share/qm1gQpz7

I only upgraded to Statamic 5, and then the frontend page, which shows an icon, throws this error. In the backend, it is working okay.

**Comment by eminos** [comment]
Oh ok. I see. But that's a different error message from what you wrote in the original post :)
This will be fixed in the next update coming in a few minutes.

**Comment by eminos** [comment]
This should be fixed in v1.3.0. Thanks for reporting!

**Comment by plompd** [comment]
It seemed the same error. Didn't take a good look at it. Thanks for the quick fix!
